substrate/frame/vesting/src/tests.rs

```diff
@@ -630,8 +633,10 @@ fn merge_ongoing_schedules() {
 
 		let sched1 = VestingInfo::new(
 			ED * 10,
-			ED,                          // Vest over 10 blocks.
-			sched0.starting_block() + 5, // Start at block 15.
+			// Vest over 10 blocks.
+			ED,
+			// Start at block 15.
+			sched0.starting_block() + 5,
 		);
 		assert_ok!(Vesting::vested_transfer(Some(4).into(), 2, sched1));
 		assert_eq!(VestingStorage::<Test>::get(&2).unwrap(), vec![sched0, sched1]);
```

Comment:
it would be good to have one test with a call to `vested_transfer` implementation.

Reply:
done